### ISSUE TITLE
INSTALLATION.md not found

### ISSUE THREAD

**Issue by shiyuwang-jamk** [JimmyToluene/GPT_ADHDer_Learning_prompt#1]
The URL in README.md: https://github.com/JimmyToluene/GPT_ADHDer_Learning_prompt/blob/main/INSTALLATION.md does not work.

**Comment by JimmyToluene** [comment]
Yes because this project just initiated on my class.There still need so
many implementations
